### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by BigMorty**
## Purpose

Update readme to...
* Instruct user to install functions extension
* State which project to select as default when adding .vscode artifacts

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code

```
see Readme for details
```

## What to Check
Verify that the following are valid
* Verify endpoints work

### CODE REVIEW

File: starters/http/dotnet/README.md [hunk 1]
```diff
@@ -101,9 +101,10 @@ content-type: application/json
 
 1) Open this folder in a new terminal
 2) Open VS Code by entering `code .` in the terminal
-3) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete
-4) Press Run/Debug (F5) to run in the debugger
-5) Use same approach above to test using an HTTP REST client
+3) Make sure the Azure Functions extension is installed
```
Comment (mattchenderson) [inline starters/http/dotnet/README.md:104]: Link to extension would be nice
```suggestion
3) Make sure the [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) is installed
```

File: starters/http/dotnet/README.md [hunk 2]
```diff
@@ -101,9 +101,10 @@ content-type: application/json
...
+4) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete (select `starters\http\dotnet\http` project when prompted to set a default)
```
Comment (mattchenderson) [inline starters/http/dotnet/README.md:105]: Probably need to tell them how to open this. `Crtl+Shift+P` / `Cmd+Shift+P` and then type in that command.

Comment (BigMorty) [reply starters/http/dotnet/README.md]: done

File: starters/http/dotnet/README.md [hunk 3]
```diff
@@ -101,9 +72,10 @@ content-type: application/json
 
 1) Open this folder in a new terminal
 2) Open VS Code by entering `code .` in the terminal
-3) Add a **.vscode** folder by running *"Azure Functions: Initialize project for use with VS Code"* in the Command Pallete
-4) Press Run/Debug (F5) to run in the debugger
-5) Use same approach above to test using an HTTP REST client
+3) Make sure the [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) is installed
+4) Add needed files to the **.vscode** folder by opening the command palette (`Crtl+Shift+P/Cmd+Shift+P` and selecting *"Azure Functions: Initialize project for use with VS Code"* (select `starters\http\dotnet\http` project when prompted to set a default)
```
Comment (paulyuk) [inline starters/http/dotnet/README.md:76]: Something got munged here, where I see CTL SHIFT P twice, and two open ('s.

**Review by paulyuk — CHANGES_REQUESTED** [review]
I like the streamlining and adds here.  Please see comment - something got munged in a step.

File: starters/http/dotnet/README.md (hunk 3 above)
Comment (BigMorty) [reply starters/http/dotnet/README.md]: fixed

**Pull request merged**